.github/workflows/create_baseline.yaml

```
@@ -0,0 +1,46 @@
+name: "CovidHub-baseline"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 19 * * 3"
```

Maybe a bit later in case the data posts late? This is 19:30 UTC and we don't really need the baseline much before the deadline (23:00 US/Eastern ==> 03:00 or 04:00 UTC the next day, depending on daylight savings). Maybe a bit before the end of the US/Eastern workday, e.g. 20:30 UTC

```suggestion
    - cron: "30 20 * * 3"
```